annotation/tests/test_annotations.py

```diff
@@ -0,0 +1,130 @@
+import uuid
+from datetime import datetime
+from unittest.mock import Mock
+
+import pytest
+
+from annotation.annotations.main import row_to_dict
+from annotation.models import AnnotatedDoc, Category, File, Job, User
+from annotation.schemas.categories import CategoryTypeSchema
+from annotation.schemas.jobs import JobTypeEnumSchema, ValidationSchema
+from tests.override_app_dependency import TEST_TENANT
+
+
+@pytest.fixture(scope="function")
+def specific_date_and_time():
+    SPECIFIC_DATE_TIME = datetime(2024, 1, 1, 10, 10, 0)
+    yield SPECIFIC_DATE_TIME
```

Comment:
do we really need this fixture? Seems excessive. You can simply use `datetime(2024, 1, 1, 10, 10, 0)` in any test